README.md

```diff
@@ -129,5 +102,8 @@ For [CNCF project proposal process](https://github.com/cncf/toc/blob/main/proces
 create a
 new [security review issue](https://github.com/cncf/tag-security/issues/new?assignees=&labels=assessment&template=security-assessment.md&title=%5BAssessment%5D+Project+Name)
 with a
-[self-assessment](/community/assessments/guide/self-assessment.md)
-.
+[self-assessment](/community/assessments/guide/self-assessment.md).
+
+## TAG Emeritus Leaders
+
+A big thank you to all the [tag emeritus leaders](/tag-emeritus-leaders.md) of this TAG! Your hard work and dedication have helped to make this project a success. Your valuable contributions have enabled us to develop a strong contributor strategy and build a thriving open-source community. Thank you for all that you have done!
```

Comment:
The statement in this section feels overly sentimental and doesn't add any value. It is also not consistent with the tone of the rest of the README. Consider removing or revising it to be more concise and focused, especially since leadership is already linked elsewhere.

Reply:
Good point, agreed.